lib/tbot/config/output_identity.go

```diff
@@ -126,6 +152,15 @@ func (o *IdentityOutput) CheckAndSetDefaults() error {
 		)
 	}
 
+	switch o.SSHConfigMode {
+	case SSHConfigModeNone:
+		log.DebugContext(context.TODO(), "Defaulting to SSHConfigModeOn")
```

Comment:
nit: unless you are planning a follow up, avoid context.TODO().

```suggestion
		log.DebugContext(context.Background(), "Defaulting to SSHConfigModeOn")
```

Reply:
👍